storage/database/db_manager.go

```diff
@@ -478,6 +522,39 @@ func (dbm *databaseManager) SetStateTrieMigrationDB(blockNum uint64) {
 		logger.Crit("Killed the process due to the failure of storing the migration status")
 	}
 	logger.Info("Finished setting a new database for state trie migration")
+	return nil
+}
+
+// FinishStateMigration updates stateTrieDB and triggers to remove old one.
+func (dbm *databaseManager) FinishStateMigration() {
+	oldDBDir := dbm.getDBDir(StateTrieDB)
+	newDBDir := dbm.getDBDir(StateTrieMigrationDB)
+	dbm.setDBDir(StateTrieDB, newDBDir)
```

Comment:
```suggestion
	dbm.setDBDir(StateTrieDB, newDBDir)
	dbm.setDBDir(StateTrieMigrationDB, "")
```

Reply:
I reflected in https://github.com/klaytn/klaytn/pull/502